pages/teaser.tsx

```diff
@@ -0,0 +1,76 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import styles from '../styles/Home.module.css'
+import { HamburgerMenu } from '../components/hamburger_menu'
+
+const Home: NextPage = () => {
+  return (
+    <div className="bg-noble-white">
+      <HamburgerMenu></HamburgerMenu>
```

Comment:
子要素がない場合は閉じタグを分けなくておｋ

```suggestion
      <HamburgerMenu />
```